src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -135,6 +136,9 @@ private ProgressStepDefinition[] CreateWorkflowSteps(IProgressController control
                 new ProgressStepDefinition(null, StepAttributes.Hidden | StepAttributes.Indeterminate,
                         (token, notifications) => this.FinishSolutionBindingOnUIThread(controller, token)),
 
+                new ProgressStepDefinition(Strings.BindingProjectsDisplayMessage, StepAttributes.BackgroundThread | StepAttributes.Indeterminate,
```

Comment:
What does Indeterminate flag do?

Reply:
It's means the progress is indeterminate i.e. can't be reported as a percentage/step x of y. So the UI will show something like a spinner to indicate work is happening, but no indication of how much there is left to do.